### ISSUE TITLE
missing visualisation of sentences with no matching node with optional selector

### ISSUE THREAD

**Issue by Ansa211** [ufal/perl-pmltq-web#80]
Compare the output of the following two queries - they differ by a single dot on the third line of the query:
http://hdl.handle.net/11346/PMLTQ-6UJY
http://hdl.handle.net/11346/PMLTQ-5D3K

With the second query, the tree corresponding to the found sentence is not displayed.

**Comment by matyaskopp** [comment]
Problem occurs when optional selector is used

http://hdl.handle.net/11346/PMLTQ-L7VX


```json
{  
   "nodes":[  
      [ "a", "t-node" ],
      [ "b", "t-node" ],
      [ "g", "a-node" ]
   ],
   "results":[  
      [  
         "353\/t-node@t-cmpr9410-019-p10s2w8",
         "356\/t-node@t-cmpr9410-019-p10s2w11",
         "430\/a-node@a-cmpr9410-019-p10s2w9"
      ],
      [  
         "869\/t-node@t-cmpr9410-019-p26s2w24",
         "870\/t-node@t-cmpr9410-019-p26s2w23",
         null
      ],
...
   ]
}
```

and no node is matched (second result: `null`). Function `parseNodeName(name)` with parameter `name=null` craches and no tree is rendered (the tree of previous result is used).

**Comment by matyaskopp** [comment]
fixed with  bf2e50e
